tools/travis/osx_run.sh

```diff
@@ -2,12 +2,22 @@
 
 set -e -x
 
-shell_session_update() { :; }
+shell_session_update() { 
+  set +e
+  echo "This is so travis doesn't break"
+  return 0;
+}
+
+rvm get head
```

Comment:
I think that everything that is related to Travis compilation should be done in another PR.

Reply:
will be, travis is slow so I was testing multiple things at once

Reply:
Ok!